deps/patches/config.sub

```diff
@@ -0,0 +1,1791 @@
+#! /bin/sh
+# Configuration validation subroutine script.
+#   Copyright 1992-2018 Free Software Foundation, Inc.
+
+timestamp='2018-12-31'
+
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
```

Comment:
Oof, yuck, can we make a note somewhere to get rid of this file at some point so that we don't indefinitely carry around another GPL dependency?

Reply:
Well, it's the same file that comes with out dependencies and also there's a license exception below. In any case, we don't distribute it with our binary distribution though, so it doesn't affect the license of that.

Reply:
Okay cool